src/PolykeyAgent.ts

```diff
@@ -494,7 +495,7 @@ class PolykeyAgent {
         async (keyChangeData: RootKeyPairChangeData) => {
           this.logger.info('Propagating root keypair change');
           await this.status.updateStatusLive({
-            nodeId: keyChangeData.nodeId,
+            nodeIdEncoded: nodesUtils.encodeNodeId(keyChangeData.nodeId),
```

Comment:
Can you keep the key as `nodeId` here? The `status.json` keys should not change.

Reply:
This has been changed back.